FastBank.Services/TransactionService/TransactionService.cs

```diff
@@ -70,30 +295,20 @@ public void TransactionReport(User user)
 
                 _menuService.ShowLogo();
 
-                Console.WriteLine("{0,-25} {1,-10} {2,-30} {3, -40}",
-                "| Date",
-                "| Amount",
-                "| Transaction Type",
-                "| Customer email");
-                Console.WriteLine(new string('-', 85));
-
-                foreach (var transaction in transactions)
-                {
-                    Console.WriteLine("{0,-25} {1,-10} {2,-30} {3, -40}",
-                        $"| {transaction.CreatedDate}",
-                        $"| {transaction.Amount}",
-                        $"| {transaction.TransactionType.GetDisplayName()}",
-                        $"| {transaction.CreatedByUser.Email}");
-                }
+                ShowTransactionList(transactions);
 
                 Console.WriteLine($"\nPage {currentPage}/{totalPages}\n");
 
+                var transactionsReports = ShowLastTransactionsReports(user);
+
```

Comment:
Fix variable and method name as mentioned how i want them to be named in above comments